inventory/main/src/main/java/org/opennms/horizon/inventory/grpc/MonitoringSystemGrpcService.java

```diff
@@ -115,4 +128,11 @@ private Status createStatusNotExist (String systemId) {
             .setMessage("Monitor system with system id: " + systemId + " doesn't exist")
             .build();
     }
+
+    private Status createStatusNotExist (long systemId) {
+        return Status.newBuilder()
+            .setCode(Code.NOT_FOUND_VALUE)
+            .setMessage("Monitor system with system id: " + systemId + " doesn't exist")
+            .build();
+    }
```

Comment:
Is this supposed to be `systemId`, or `id`? The `long` type implies `id`.

```suggestion
    private Status createStatusNotExist (long id) {
        return Status.newBuilder()
            .setCode(Code.NOT_FOUND_VALUE)
            .setMessage("Monitor system with id: " + id + " doesn't exist")
            .build();
    }
```